src/middleware/jirahost-middleware.ts

```diff
@@ -68,7 +74,7 @@ export const jirahostMiddleware = async (req: Request, res: Response, next: Next
 			next();
 		});
 	} else {
-		res.locals.jiraHost = req.session.jiraHost;
+		res.locals.jiraHost = req.session.jiraHost as string;
```

Comment:
This could be some potential bug.
So at runtime, `jiraHostMiddleware` can actually have an empty `res.locals.jiraHost` if session is empty.

But lots of routes after this middleware just assume the `jiraHost` is always there. 

--

This only fix the ts complaints, doesn't fix anything at runtime. Might need to look at it later.

Reply:
This is on purpose.  It's suppose to be undefined if it's missing.  there's a middleware that catches the issue of missing jirahost so that all others can assume it works.

Reply:
`there's a middleware that catches the issue of missing jirahost` <-- can you tell me which one is it? I must over looked it.